ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -692,6 +692,12 @@ func (r *RayClusterReconciler) reconcilePods(ctx context.Context, instance *rayv
 		deleted := struct{}{}
 		numDeletedUnhealthyWorkerPods := 0
 		for _, workerPod := range workerPods.Items {
+			logger.Info("reconcilePods", "Worker Pod Name: ", workerPod.Name, " Status: ", workerPod.Status.Phase)
+			if workerPod.Status.Phase != "Running" && workerPod.Status.Message != "" {
+				logger.Info("reconcilePods", "Worker Pod Name: ", workerPod.Name, " Status: ", workerPod.Status.Phase, " Message: ", workerPod.Status.Conditions[0].Message)
+			} else {
+				logger.Info("Pods created for RayCluster")
```

Comment:
This log message is unclear to me? Is this covering a different condition like `workerPod.Status.Phase == Created`? If so, I think it's a good idea to make this explicit

Reply:
Thanks for the feedback, I will replace the 'else' condition with something more explicit

Reply:
I think it's better to remove the 'else' condition since the status is already printed above